certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -3927,68 +3939,591 @@
+                },
+                "IdleThresholdInSeconds": {
+                  "type": "integer",
+                  "format": "int32",
+                  "default": 120,
+                  "title": "Idle threshold in seconds",
+                  "description": "The number of seconds that should have elapsed since the last input before moving the mouse."
+                },
+                "IdleCheckPeriodInSeconds": {
+                  "type": "integer",
+                  "format": "int32",
+                  "default": 30,
+                  "title": "Idle check period in seconds",
+                  "description": "The number of seconds between checking the last input time (to check if the idle threshold has been passed)."
+                },
+                "TotalKeepaliveRuntimeInSeconds": {
+                  "type": "integer",
+                  "format": "int32",
+                  "default": -1,
+                  "title": "Total keepalive runtime in seconds",
+                  "description": "The total number of seconds to keep the session alive for."
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}
+          }
+        },
+        "operationId": "KeepSessionAlive"
+      }
+    },
+    "/Environment/StopKeepSessionAlive": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Stop keep session alive",
+        "description": "Stops the IA-Connect Agent user session keep alive.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "StopKeepSessionAlive",
+            "description": "Inputs for IA-Connect StopKeepSessionAlive action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL  Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Updated output schema as requested.